viz_scripts/energy_calculations.ipynb

```diff
@@ -244,58 +162,38 @@
     "eirc = eirc.sort_values(by=['Sketch of Total Energy_Impact(kWH)'], ascending=False)\n",
     "eirc['boolean'] = eirc['Sketch of Total Energy_Impact(kWH)'] > 0\n",
     "\n",
+    "subset1 = eirc [['Replaced_mode', 'Sketch of Total Energy_Impact(kWH)']].copy()\n",
+    "subset1.rename(columns = {'Replaced_mode':'Transport Mode','Sketch of Total Energy_Impact(kWH)':'Replaced_Mode' }, inplace=True)\n",
+    "\n",
     "#eimc : energy impact mode_confirm\n",
     "eimc=expanded_ct.groupby('Mode_confirm').agg({'Energy_Impact(kWH)': ['sum', 'mean']},)\n",
     "eimc.columns = ['Sketch of Total Energy_Impact(kWH)', 'Sketch of Average Energy_Impact(kWH)']\n",
     "eimc = eimc.reset_index()\n",
     "eimc = eimc.sort_values(by=['Sketch of Total Energy_Impact(kWH)'], ascending=False)\n",
     "\n",
-    "\n",
-    "subset1 = eirc [['Replaced_mode', 'Sketch of Total Energy_Impact(kWH)']].copy()\n",
-    "subset1.rename(columns = {'Replaced_mode':'Transport Mode','Sketch of Total Energy_Impact(kWH)':'Replaced_Mode' }, inplace=True)\n",
-    "\n",
     "subset2 = eimc [['Mode_confirm', 'Sketch of Total Energy_Impact(kWH)']].copy()\n",
     "subset2.rename(columns = {'Mode_confirm':'Transport Mode','Sketch of Total Energy_Impact(kWH)':'Mode_Confirm' }, inplace=True)\n",
     "\n",
     "df_plot = pd.merge(subset1, subset2, on=\"Transport Mode\")\n",
     "df = pd.melt(df_plot , id_vars=['Transport Mode'], value_vars=['Replaced_Mode','Mode_Confirm'], var_name='selection')\n",
-    "df.rename(columns = {'value':'Energy Impact (kWH)'}, inplace = True)"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "mechanical-error",
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "df"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "unknown-venice",
-   "metadata": {
-    "scrolled": false
-   },
-   "outputs": [],
-   "source": [
+    "df.rename(columns = {'value':'Energy Impact (kWH)'}, inplace = True)\n",
+    "\n",
     "df= df.sort_values(by=['Energy Impact (kWH)'], ascending=False)\n",
     "x= 'Energy Impact (kWH)'\n",
     "y= 'Transport Mode'\n",
     "color = 'selection'\n",
     "plot_title=\"Sketch of Energy Impact (kWH) by Transport Mode\\n%s\" % quality_text\n",
-    "file_name ='sketch_all_energy_impact%s.png' % file_suffix\n",
-    "overeall_energy_impact(x,y,color,df,plot_title,file_name)"
+    "file_name ='sketch_all_energy_impact%s' % file_suffix\n",
+    "overeall_energy_impact(x,y,color,df,plot_title,file_name)\n",
```

Comment:
<img width="862" alt="image" src="https://user-images.githubusercontent.com/5828311/188195222-1400335c-0750-421e-a887-c2ed76a61d42.png">

I think this graph is a stacked barplot version of the "sketch_all_mode_energy_impact" plot.

Reply:
Happy to remove or keep it, I am not entirely sure I understand it

Reply:
We should remove it because it is confusing - it is not just you :) and we won't have the replaced mode for non-modes-of-interest in the future. But I am not going to hold up the merge for this. 

Added to https://github.com/e-mission/e-mission-docs/issues/788